docs/src/examples/components/Icon/Variations/IconExampleColor.shorthand.tsx

```diff
@@ -3,10 +3,42 @@ import { Icon } from '@stardust-ui/react'
 
 const IconExampleColor = () => (
   <div>
-    <Icon name="home" />
-    <Icon name="home" variables={{ color: 'blue' }} />
-    <Icon name="home" variables={{ color: 'red' }} />
-    <Icon name="home" variables={{ color: 'orange' }} />
+    {/* The colors for the icon and the border are inherited. */}
+    <div style={{ color: 'violet', padding: '1rem', display: 'inline-block' }}>
```

Comment:
May be we can use our `Grid` component to avoid styling (`display`, `padding`)?

```jsx
  <Grid columns='repeat(4, auto)'>
    <div />
    <div />
    // others
  </Grid>
```

Reply:
We can also split this to 4 four examples and may be this will better option. For example, https://react.semantic-ui.com/collections/menu/#types-pointing

This section has two examples and we can do the same.

Reply:
Yep, it makes sense to have four different examples. Will refactor this. Thanks!

Reply:
We decided offline, that we will use the Grid with description for each of the examples, as introducing another sections will introduce unnecessary additional scrolling on the page.